sdk/formrecognizer/ai-form-recognizer/review/ai-form-recognizer.api.md

```diff
@@ -0,0 +1,1299 @@
+                    };
+                };
+                readonly WorkPhones: {
+                    readonly type: "array";
+                    readonly items: {
+                        readonly type: "phoneNumber";
+                        readonly description: "Work phone number";
+                        readonly example: "+1 (987) 213-5674";
+                    };
+                };
+                readonly MobilePhones: {
+                    readonly type: "array";
+                    readonly items: {
+                        readonly type: "phoneNumber";
+                        readonly description: "Mobile phone number";
+                        readonly example: "+1 (987) 123-4567";
+                    };
+                };
+                readonly Faxes: {
+                    readonly type: "array";
+                    readonly items: {
+                        readonly type: "phoneNumber";
+                        readonly description: "Fax number";
+                        readonly example: "+1 (987) 312-6745";
+                    };
+                };
+                readonly OtherPhones: {
+                    readonly type: "array";
+                    readonly items: {
+                        readonly type: "phoneNumber";
+                        readonly description: "Other phone number";
+                        readonly example: "+1 (987) 213-5673";
+                    };
+                };
+                readonly Emails: {
+                    readonly type: "array";
+                    readonly items: {
+                        readonly type: "string";
+                        readonly description: "Contact email";
+                        readonly example: "chris.smith@contoso.com";
+                    };
+                };
+                readonly Websites: {
+                    readonly type: "array";
+                    readonly items: {
+                        readonly type: "string";
+                        readonly description: "Website";
+                        readonly example: "https://www.contoso.com";
+                    };
+                };
+            };
+        };
+    };
+};
+
+// @public
+export interface CommonClientOptions extends PipelineOptions {
+    apiVersion?: FormRecognizerApiVersion;
+    stringIndexType?: StringIndexType;
+}
+
+// @public
+export type ContentType = "application/octet-stream" | "application/pdf" | "image/bmp" | "image/jpeg" | "image/png" | "image/tiff";
+
+// @public
+export interface CopyAuthorization {
+    accessToken: string;
+    expirationDateTime: Date;
+    targetModelId: string;
+    targetModelLocation: string;
+    targetResourceId: string;
+    targetResourceRegion: string;
+}
+
+// @public
+export interface CopyModelOptions extends OperationOptions, PollerOptions<TrainingPollOperationState> {
+}
+
+// @public
+export interface CustomDocumentModelsInfo {
+    count: number;
+    limit: number;
```

Comment:
Based on the description, did you consider a clearer name? e.g. `maxCountAllowed`?

Reply:
It's accessed as `info.customDocumentModels.count` and `info.customDocumentModels.limit`, so in context I think it's pretty clear what it means. In any case we are just exposing a generated type here. This service is in preview, so I'd like to take rename considerations to them first before we consider doing a name mapping, as it'll be best overall if the service and SDK names are aligned.